examples/2D_ibm_steady_shock/case.py

```diff
@@ -0,0 +1,176 @@
+import json
+import math
+
+Mu = 1.84E-05
+gam_a = 1.4
+gam_b = 1.1
+
+
+x0      = 10E-06
+p0      = 101325
+rho0    = 1.0
+c0      = math.sqrt( p0/rho0 )
+patm    = 1.
+rhoatm    = 1.
+
+#air props
+n_tait  = 1.4
+B_tait  = 0.0
+
+
+vf0 = 0.0 
+
+cact    =  math.sqrt(n_tait*(p0 + p0*B_tait) / ((1 - vf0) * rho0) )
+cfl     = 0.3
+Nx = 400
+Ny = 200
+dx = 6.E-03 / (x0*float(Nx))
+dt      = cfl*dx*c0/cact
+
+
+vel = 1.5
+vel_ac = vel * c0 
+
+Min = (n_tait + 1)*vel_ac + math.sqrt((n_tait + 1)**2 * vel_ac**2 + 16 * cact**2)
+Min = Min / (4 * cact)
+beta = (n_tait + 1) * Min**2 / (Min**2 * (n_tait - 1 + 2*vf0) + 2 * (1 - vf0))
+delta = (1 - vf0) + n_tait * Min**2 * (beta - 1) * (1 + B_tait) / beta
+
+vel = Min * cact * (beta - 1) / (beta * c0)
+
+# Configuring case dictionary
+print(json.dumps({
+    # Logistics ================================================================
```

Comment:
It might be worthwhile to remove the "distracting characters," as mentioned in #609, to save work for whoever fixes it down the road and avoid creating more problems.